tests/sentry/hybridcloud/tasks/test_backfill_outboxes.py

```diff
@@ -78,7 +80,7 @@ def test_region_processing(task_runner):
 
 @django_db_all
 @control_silo_test
-def test_control_processing(task_runner):
+def test_control_processing(task_runner: Callable[..., Any]) -> None:
```

Comment:
Do we have any type exemptions on these files that we need to clear now that we're explicitly typing things?

Reply:
In their new location they get opted into our standard mypy rules which led to additional types being added.